widget/ui/src/components/ExpandedRoute/ExpandedRoute.types.ts

```diff
@@ -0,0 +1,18 @@
+import type { PriceImpactProps } from '../PriceImpact/PriceImpact.types';
+import type { PropTypes as QuoteCostProps } from '../QuoteCost/QuoteCost.types';
+import type { Step } from '../StepDetails/StepDetails.types';
+import type { RouteTag } from 'rango-sdk';
+import type { SwapInputProps } from 'src/containers/SwapInput/SwapInput.types';
+
+export interface PropTypes {
+  steps: Step[];
+  tags: RouteTag[];
+  time: QuoteCostProps['time'];
+  fee: QuoteCostProps['fee'];
+  percentageChange?: PriceImpactProps['percentageChange'];
+  warningLevel?: PriceImpactProps['warningLevel'];
+  outputPrice: SwapInputProps['price'];
```

Comment:
I'm not sure. Isn't better to move this component to widget embedded component and just pass `quote: SelectedQuote` as a props?

```
export interface PropTypes {
  quote: SelectedQuote
  tooltipContainer?: HTMLElement;
  onClick?: () => void;
  selected?: boolean;
}
```